apps/framework-cli/src/infrastructure/olap/clickhouse_alt_client.rs

```diff
@@ -0,0 +1,194 @@
+use std::str::FromStr;
+use std::time::Duration;
+
+use chrono::{DateTime, Days, NaiveDate};
+use clickhouse_rs::errors::{Error, FromSqlError};
+use clickhouse_rs::types::{ColumnType, Row};
+use clickhouse_rs::types::{FromSql, FromSqlResult, Options, ValueRef};
+use clickhouse_rs::ClientHandle;
+use futures::stream::BoxStream;
+use futures::StreamExt;
+use serde::__private::from_utf8_lossy;
+use serde_json::{json, Map, Value};
+use swc_common::pass::Either;
+
+use crate::framework::data_model::schema::EnumValue;
+use crate::infrastructure::olap::clickhouse::config::ClickHouseConfig;
+use crate::infrastructure::olap::clickhouse::model::{ClickHouseColumnType, ClickHouseTable};
+
+pub fn get_pool(click_house_config: &ClickHouseConfig) -> clickhouse_rs::Pool {
+    let address = format!(
+        "tcp://{}:{}",
+        click_house_config.host, click_house_config.native_port
+    );
+
+    clickhouse_rs::Pool::new(
```

Comment:
Does this create a new pool every time? or does it get a pool based on the `address`?

Reply:
https://github.com/514-labs/moose/pull/1198/files#diff-d4af486162b78407e0eea7401a6a4376ac6e0578a5f8a70144a06be8549c0519R240